.eslintignore

```diff
@@ -1,3 +1,4 @@
 coverage
 site
 node_modules
+package-lock.json
```

Comment:
Since we're linting JSON files as well, we want to ignore this auto-generated beast since it's huge and slow (and pointless) to lint.